lib/generators/audited/templates/create_audit_associations.rb

```diff
@@ -0,0 +1,26 @@
+class <%= migration_class_name %> < <%= migration_parent %>
+  def self.up
+    create_table :audit_associations, :force => true do |t|
+      t.column :audit_id, :integer
+      t.column :associated_id, :integer
+      t.column :associated_type, :string
+    end
+
+    execute <<-SQL
+      INSERT INTO audit_associations (audit_id, associated_id, associated_type)
+        SELECT id, associated_id, associated_type
+        FROM audits
+    SQL
+
+    add_index :audit_associations, :audit_id, :name => 'index_audit_associations_on_audit_id'
+    add_index :audit_associations, [:associated_type, :associated_id], :name => 'index_audit_associations_on_associated'
+
+    remove_index :audits, name: 'associated_index'
```

Comment:
We're going to have to run some similar migration here in web, right? (I assume we don't invoke this generator to do so, but I could be wrong). 

This table seems to already be pretty damn big, so dropping indices and columns makes me kinda nervous.

Reply:
@rtroxler we'll invoke the `upgrade_generator` which will create this file.

If we want we can handle the actual way we do it differently in web.

Reply:
Yeah I feel like we might want to ... maybe not drop these columns since that'll lock that table and probably cause downtime if I had to guess since so many requests will be writing to it now.